plugins/web-worker-offloading/third-party/woocommerce.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Web Worker Offloading integration with WooCommerce.
+ *
+ * @since 0.1.0
+ * @package web-worker-offloading
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Configures WWO for WooCommerce and Google Analytics.
+ *
+ * @since 0.1.0
+ * @access private
+ * @link https://partytown.builder.io/google-tag-manager#forward-events
+ *
+ * @param array<string, mixed>|mixed $configuration Configuration.
+ * @return array<string, mixed> Configuration.
+ */
+function wwo_woocommerce_configure( $configuration ): array {
+	$configuration = (array) $configuration;
+
+	$configuration['mainWindowAccessors'][] = 'wp';   // Because woocommerce-google-analytics-integration needs to access wp.i18n.
+	$configuration['mainWindowAccessors'][] = 'ga4w'; // Because woocommerce-google-analytics-integration needs to access window.ga4w.
+	$configuration['globalFns'][]           = 'gtag'; // Because gtag() is defined in one script and called in another.
+	$configuration['forward'][]             = 'dataLayer.push'; // Because the Partytown integration has this in its example config.
```

Comment:
Don't these cause PHP notices because the keys they add to are not initialized as arrays?

Reply:
No it doesn't: https://3v4l.org/YPITA

Reply:
In fact, this also doesn't cause any notice:

```php
$foo['bar']['baz'][] = 'quux';
```

It results in `$foo` being defined as:

```
array(1) {
  ["bar"]=>
  array(1) {
    ["baz"]=>
    array(1) {
      [0]=>
      string(4) "quux"
    }
  }
}
```

https://3v4l.org/rMVi7

Reply:
Hmm, I learned something today. :)